### ISSUE TITLE
Investigate PBN, topic, organization, and resource API for data import.

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#562]
_From @aroubin on October 26, 2018 19:54_

Investigate PBN APIs and the topic, organization, and resource data that is available for HI. 

https://www.lawhelp.org/hi/api/v2/resources
https://www.lawhelp.org/hi/api/v2/organizations
https://www.lawhelp.org/hi/api/v2/organizations?list=detail
https://www.lawhelp.org/hi/api/v2/topics?list=detail

_Copied from original issue: Microsoft/Access2Justice#687_

**Comment by aroubin** [comment]
_From @amitabhm21 on December 7, 2018 23:24_

The timelines for this task is beyond the scope of MSFT contribution, Closing this issue,

**Comment by KellyeDC-zz** [comment]
@breeglej @grawdon Is this in scope?  Are we still going in this direction for Hawaii static content?

**Comment by grawdon** [comment]
No, this is not in scope. Content from PBN would be listed as URL, not imported to LN. LawHelp states list resource providers but the only integration for this being discussed is for 211.

**Comment by breeglej** [comment]
Agree with Glenn.  This is not in scope.
